src/typescript/azure-openapi-validator/rules/Rpaas_CreateOperationAsyncResponseValidation.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from "../rule"
+export const Rpaas_CreateOperationAsyncResponseValidation: string = "Rpaas_CreateOperationAsyncResponseValidation"
+
+rules.push({
+  id: "R4023",
+  name: Rpaas_CreateOperationAsyncResponseValidation,
+  severity: "error",
+  category: "RPaaSViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.rpass,
+  appliesTo_JsonQuery: "$.paths.*.put",
+  *run(doc, node, path) {
+    if (node.responses["202"]) {
+      yield {
+        message: `[RPaaS] Only 201 is the supported response code for PUT async response.`,
+        location: path
```

Comment:
this path could contains 202 , so that the location would be more accurate

Reply:
$paths[].put["responses"].202

Reply:
replace to path.concat(["responses","202"])

Reply:
Thanks

Reply:
thanks. updated for other one too.